observability/otel/otel-collector/otel-collector-extensions.adoc

```diff
@@ -446,18 +445,35 @@ include::snippets/technology-preview.adoc[]
         protocols:
           http: {}
     exporters:
-      otlp:
+      debug:
 
     service:
       extensions: [zpages]
       pipelines:
         traces:
           receivers: [otlp]
-          exporters: [otlp]
+          exporters: [debug]
 # ...
 ----
 
-<1> Specifies the HTTP endpoint that serves zPages. Use `localhost:` to make it available only locally, or `":"` to make it available on all network interfaces. The default is `localhost:55679`.
+<1> Specify the HTTP endpoint for serving the zPages extension. The default is `localhost:55679`.
+
+The {OTELOperator} does not expose this route. To access the HTTP endpoint, forward the port:
+
+[source,terminal]
+----
+$ oc port-forward pod/$(oc get pod -l app.kubernetes.io/name=instance-collector -o=jsonpath='{.items[0].metadata.name}') 55679
+----
+
+To get the zPages information, you can visit `http://localhost:55679/debug/servicez` in your browser.
```

Comment:
The link will not work for you because you need to execute all the commands described previously.
It is not an additional resource; it becomes available when the user runs the previous commands. This URL needs to be documented clearly.

Reply:
Thanks for clarifying, @iblancasa . This is where the REFERENCE module gets tricky as procedural steps are being discussed. As annoying as this is, maybe having separate procedure and reference module would make for a more comprehensive understanding of what steps are needed to reach an end goal?

Reply:
I think I can offer a suggestion to rewrite this in a more referential tone.

Reply:
See https://redhat-documentation.github.io/modular-docs/#reference-module-guidelines . A CONCEPT AND REFERENCE module could also work, but instead of procedure steps, you could write a scenario based on an example.

Reply:
@iblancasa and @dfitzmau, here is my suggestion:
https://github.com/openshift/openshift-docs/pull/82621/files#r1782949553